lib/web_ui/dev/test_platform.dart

```diff
@@ -197,7 +197,9 @@ class BrowserPlatform extends PlatformPlugin {
         'golden_files',
       );
     } else {
-      await fetchGoldens();
+      if(!isLuci || !Platform.isMacOS) {
```

Comment:
Let's leave a comment explaining why these are special-cased.

Also nit: add one space between `if` and `(`.

Reply:
done.